tools/src/main/scala/scala/scalanative/build/IO.scala

```diff
@@ -89,40 +134,79 @@ private[scalanative] object IO {
     val stream       = Files.newInputStream(path)
     val digestStream = new DigestInputStream(stream, digest)
     val buf          = new Array[Byte](bufSize)
```

Comment:
The `val buf =` should also be inside the `try`.

Reply:
I see why, thank-you.